pkg/engine/engine_dcos.go

```diff
@@ -0,0 +1,250 @@
+//+build !test
```

Comment:
@mboersma I was hoping that this pragma would remove this file from unit test coverage calculation, which should have moved the needle on our % coverage by a wee bit. Do you know of any better ideas?

Reply:
You could add it to the `ignore:` section in .codecov/codecov.yml.

Reply:
does `//+build !test` do anything?

Reply:
It tells the compiler not to include this when running unit tests

Reply:
but does it actually work? I understood your comment above to mean that it doesn't

Reply:
It didn't seem to influence our current test coverage toolchain, but I'm not sure that our current test coverage calculation implementation is vanilla go-idiomatic. So basically: I left this in place as a way of formally declaring intent for these funcs in a go-idiomatic fashion. So at minimum it's basically a code comment at this point.